.github/workflows/build.yml

```diff
@@ -11,6 +11,7 @@ on:
 jobs:
     tests:
         runs-on: ${{ matrix.os }}
+        continue-on-error: ${{ matrix.experimental }}
```

Comment:
this is not the right approach for making experimental builds: this would continue the job even if `setup-php` fails for instance.

Reply:
According to the [GithubAction docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error) this setting allows a job to "pass" even though one step failed. The job will still be marked as failed but the whole workflow will not fgail due to that one failed step.

If you have a different way of achieving this I am all ears!

Reply:
oh, it looks like they have separated the `continue-on-error` at the job level and at the step level now.